collection/endpoint.csv

```diff
@@ -1 +1,5 @@
 endpoint,endpoint-url,parameters,plugin,entry-date,start-date,end-date
+710664365802fd3664adb099080a4b996834888deb34ebd4c62c40cbd0e45776,https://environment.data.gov.uk/arcgis/rest/services/EA/FloodMapForPlanningRiversAndSeaFloodZone2/MapServer/0,,arcgis,2023-08-23T10:10:49:32Z,,

```

Comment:
Did you. attempt to pull out a GeoJSON from arcGIS?

Reply:
Again maybe not for now

Reply:
The downloadable format in the arcgis link was json which I tried consuming directly but it was not processing. I ll try maybe later again.